shapelib.vapi

```diff
@@ -0,0 +1,239 @@
+/* This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+*/
+
+[CCode (cprefix = "", lower_case_cprefix = "", cheader_filename = "shapefil.h")]
+namespace ShapeFil {
```

Comment:
Could this be named `Shapelib`? Namespaces in Vala are are suited for including symbols from different headers transparently.